cypress/integration/group1/dashboard.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 OICR and UCSC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { contains } from 'cypress/types/jquery';
+import { Repository } from '../../../src/app/shared/openapi/model/repository';
+import { goToTab, isActiveTab, resetDB, setTokenUserViewPort } from '../../support/commands';
+
+describe('Dockstore dashboard', () => {
+  resetDB();
+  setTokenUserViewPort();
+
+  const cwlDescriptorType = 'CWL';
+  const wdlDescriptorType = 'WDL';
+  const nextflowDescriptorType = 'Nextflow';
+  //   it('can open register modals from the homepage', () => {
```

Comment:
Probably doesn't make sense to check-in commented code without a comment explaining under what circumstance it would be uncommented

Reply:
Oh yeah I commented it out to see if that was the test that was flaking, I'll just remove it since it fails >50% of the time.